src/config/routes/2.1.json

```diff
@@ -84,6 +84,14 @@
         "action": "handle",
         "verbs": ["GET"]
     },
+
+    {
+        "path": "/password/?$",
+        "controller": "PasswordController",
+        "action": "handle",

```

Comment:
Don't use the `handle` action - use a specific method as has been done for [usernameReminder](https://github.com/joindin/joindin-api/blob/master/src/config/routes/2.1.json#L95-L100)